### PR TITLE
MOSIP-32947 Resolved UI errors when in RTL

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**

### CODE REVIEW

File: pmp-reactjs-ui/src/utils/AppUtils.js [hunk 1]
```diff
@@ -115,4 +115,13 @@ export const handleServiceErrors = (responseData, setErrorCode, setErrorMsg) =>
         setErrorMsg(errorMessage);
         console.error('Error:', errorMessage);
     }
+}
+
+export const RTLStyles = (langCode) => {
```
Comment (mayuradesh) [inline pmp-reactjs-ui/src/utils/AppUtils.js:120]: rename to isLangRTL()

File: pmp-reactjs-ui/src/pages/dashboard/Dashboard.js [hunk 2]
```diff
@@ -18,6 +19,7 @@ import authServiceIcon from '../../svg/auth_services_icon.svg';
 function Dashboard() {
   const navigate = useNavigate();
   const { t } = useTranslation();
+  const arabicLang = RTLStyles(getUserProfile().langCode);
```
Comment (mayuradesh) [inline pmp-reactjs-ui/src/pages/dashboard/Dashboard.js:22]: remove arabicLang variable name
rename to isLoginLanguageRTL in all pages

File: pmp-reactjs-ui/src/nav/SideNav.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ function SideNav({ open }) {
     };
     return (
         <div className="flex font-inter bg-white h-screen z-30">
-            <div className={`h-full ${open ? "absolute inset-y-16 left-0 w-64" : "absolute inset-y-16 w-20"} 
+            <div className={`h-full ${open ? "absolute inset-y-16 w-64" : "absolute inset-y-16 w-20"}
```
Comment (mayuradesh) [inline pmp-reactjs-ui/src/nav/SideNav.js:58]: removing left-0 will this have any impact in expand collapse of side menu, scrolling of page?
check in eng as well as ara

**Review by mayuradesh — CHANGES_REQUESTED** [review]
comments added

File: pmp-reactjs-ui/src/utils/AppUtils.js (hunk 1 above)
Comment (Anil-kumar-Majji) [reply pmp-reactjs-ui/src/utils/AppUtils.js:120]: Done

File: pmp-reactjs-ui/src/pages/dashboard/Dashboard.js (hunk 2 above)
Comment (Anil-kumar-Majji) [reply pmp-reactjs-ui/src/pages/dashboard/Dashboard.js:22]: Done

**Review by mayuradesh — APPROVED** [review]

**Pull request merged**